### PR TITLE
Responsiveness 

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Guys please review for the responsiveness and hamburger in the smaller viewport .
Please review this and UI and colors I changed in this Branch
Thank you

### CODE REVIEW

File: documentation.css [hunk 1]
```diff
@@ -85,17 +99,23 @@ body {
...
     box-shadow: 0 2px 3px -1px rgb(0 0 0 / 10%);
     display: flex;
     align-items: flex-end;
     justify-content: space-between;
-    flex-wrap: wrap;
+    /* flex-wrap: wrap; */
 }
 
 .navigation-pri-part {
     display: flex;
```
Comment (ApurvaSawant11) [inline documentation.css:113]: The hamburger icon is not center-aligned. Please align it.

File: documentation.css [hunk 2]
```diff
@@ -177,6 +196,56 @@ body {
...
+
+    .universal-container {
+
+        display: grid;
+        height: 100%;
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+        grid-template-areas:
+            "aside-nav  main main main main ";
+    }
+
```
Comment (ApurvaSawant11) [inline documentation.css:214]: Add come padding on the right for this container. Rest everything looks good.

File: documentation.css (hunk 1 above)
Comment (Dev-AkshatGupta) [reply documentation.css:113]: Done with that

File: documentation.css (hunk 2 above)
Comment (Dev-AkshatGupta) [reply documentation.css:214]: Done with this too . Thanks for pointing this out :)

**Review by ParthParolekar — COMMENTED** [review]
image
The content goes out of the window in mobile screens. I would suggest adding view-port width to your main content so that it becomes responsive on every screen.

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me. Approved :)

**Comment by pradipgharge** [comment]
Bro it looks good, but not responsive on mobile screen !

**Review by VLeads — CHANGES_REQUESTED** [review]
Everything looks amazing. The thing I noticed is that in mobile, when hamburger menu is opened, it's not coming centred align rather menu is going towards right side. Thus there is shown a little margin on left but not on right.

**Review by ParthParolekar — CHANGES_REQUESTED** [review]
The whitespace in between the hamburger is not clickable. I would suggest you to wrap the hamburger inside a div and make that div clickable. The animation is great btw.

**Review by ParthParolekar — APPROVED** [review]

**Pull request merged**